tests/graphics/point_cloud_test.py

```diff
@@ -27,11 +27,26 @@ def test_update():
 
 def test_get_limits():
     da = scatter_data()
-    cloud = PointCloud(data=da, x='x', y='y', z='z')
+    pix = 0.5
+    cloud = PointCloud(data=da, x='x', y='y', z='z', pixel_size=pix)
+    xlims, ylims, zlims = cloud.get_limits()
+    assert sc.identical(xlims[0], da.meta['x'].min() - sc.scalar(0.5 * pix, unit='m'))
+    assert sc.identical(xlims[1], da.meta['x'].max() + sc.scalar(0.5 * pix, unit='m'))
+    assert sc.identical(ylims[0], da.meta['y'].min() - sc.scalar(0.5 * pix, unit='m'))
+    assert sc.identical(ylims[1], da.meta['y'].max() + sc.scalar(0.5 * pix, unit='m'))
+    assert sc.identical(zlims[0], da.meta['z'].min() - sc.scalar(0.5 * pix, unit='m'))
+    assert sc.identical(zlims[1], da.meta['z'].max() + sc.scalar(0.5 * pix, unit='m'))
```

Comment:
This is not a very useful test, maybe? This is just checking an implementation detail, and with every tiny code modification you will need to update the test.

Reply:
I'll see. The `Outline` depends on this working properly, but I agree that the usefulness factor is not through the roof.
The next time it breaks I'll remove it.